README.md

```diff
@@ -3,6 +3,30 @@
 
 ## Introduction
 
-A command line interface to interact with EdgeX microservices. Replaces the need to manually construct complex CURL commands and/or maintain developer scripts.
+EdgeX-CLI is a command line interface tool meant for developers to interact with EdgeX microservices. It replaces the need to manually construct complex cURL commands and/or maintain developer scripts.
 
-`edgex-cli` is being updated to support the V2 API and the documentation will be updated in due course. See the [EdgeXFoundry CLI documentation](https://docs.edgexfoundry.org/1.3/getting-started/tools/Ch-CommandLineInterface/) for more information about the V1.3 (Hanoi) client.
\ No newline at end of file
+See the [CLI Getting started documentation](https://docs.edgexfoundry.org/2.2/getting-started/tools/Ch-CommandLineInterface/) for more information about the client. 
+
+## Installing EdgeX-CLI
+
+The client can be installed using a [snap](https://github.com/edgexfoundry/edgex-cli/tree/main/snap):
+
+```
+sudo snap install edgex-cli
+```
+
+To build, install and run `edgex-cli` natively, do the following:
+```
+git clone http://github.com/edgexfoundry/edgex-cli.git
+cd edgex-cli
+make tidy
+make build
+./bin/edgex-cli
+```
```

Comment:
Could we also add a section here? This is based on this [ADR](https://github.com/siggiskulason/edgex-docs/blob/edgex-cli-adr/docs_src/design/adr/core/0019-EdgeX-CLI-V2.md).
## Supported commands and sub-commands

**Core Command**
- **`edgex-cli command`** `read | write | list`

**Core Data**
- **`edgex-cli event`** `add | list | rm `
- **`edgex-cli reading`** `count | list`
- **`edgex-cli config`** 

**Metadata**
- **`edgex-cli device`**  `add | adminstate | list | operstate | rm | update`
- **`edgex-cli deviceprofile`**  `add | list | rm | update`
- **`edgex-cli deviceservice`** ` add | list | rm | update`
- **`edgex-cli provisionwatcher`**  `add | list | rm | update`

**Support Notifications**
- **`edgex-cli notification`** `add | list | rm`
- **`edgex-cli subscription`** `add | list | rm`
- **`edgex-cli config`** 

**Support Scheduler**
- **`edgex-cli interval`** `add | list | rm | update`
- **`edgex-cli intervalaction`** `get | list | rm | update`
- **`edgex-cli transmission`** ` list | rm `

**Common endpoints in all services**
- **`edgex-cli version`**
- **`edgex-cli ping`**
- **`edgex-cli metrics`**

Reply:
I would rather keep this read me short and put this in the getting started documentation, which I'm working on. Otherwise we end up with documentation in two places and have to keep both up to date. Does that sound ok?

Reply:
Sounds great! Thank you.